### PR TITLE
add publications and articles to documentation

### PR DESCRIPTION
**Pull request by mariaboerner1987**

### CODE REVIEW

File: doc/source/index.rst [hunk 1]
```diff
@@ -67,6 +67,7 @@ About
    people
    organizations
    publications
+   blog-post
```
Comment (danieljanes) [inline doc/source/index.rst:70]: The matching file is missing from this PR

**Review by danieljanes — CHANGES_REQUESTED** [review]

File: doc/source/index.rst (hunk 1 above)
Comment (mariaboerner1987) [reply doc/source/index.rst:70]: push file

File: doc/source/blog-post.rst [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Blog Articles
```
Comment (danieljanes) [inline doc/source/blog-post.rst:1]:
```suggestion
Blog Posts
```

File: doc/source/blog-post.rst [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+================
+
+There are various articles writing about the usage of Flower and/or federated learning. Different useful articles will be listed here.
+If you happen to know about new articles which are not yet listed feel free to write us. We are happy
+to list all useful articles which used Flower in some form.
+
+Articles
```
Comment (danieljanes) [inline doc/source/blog-post.rst:8]:
```suggestion
Posts
```

File: doc/source/blog-post.rst [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+----------------
```
Comment (danieljanes) [inline doc/source/blog-post.rst:9]:
```suggestion
-----
```

File: doc/source/blog-post.rst [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+- Flower Blog: `Learn to adapt Flower to your Use Case <https://flower.dev/blog>`_
```
Comment (danieljanes) [inline doc/source/blog-post.rst:11]:
```suggestion
- Official Flower Blog: `Learn to adapt Flower for your use-case <https://flower.dev/blog>`_
```

File: doc/source/publications.rst [hunk 6]
```diff
@@ -9,3 +9,5 @@ Papers
 --------------
 
 - `Flower: A Friendly Federated Learning Research Framework <https://arxiv.org/abs/2007.14390>`_
+- `A first look into the carbon footprint of federated learning <https://arxiv.org/abs/2102.07627>`_
+- `On-device Federated Learning with Flower <https://arxiv.org/abs/2104.03042>`_
```
Comment (danieljanes) [inline doc/source/publications.rst:13]: @mariaboerner1987 please check where the Flower paper is cited and add those papers that are based on Flower (but not the ones that merely mention Flower alongside other frameworks)

**Review by danieljanes — CHANGES_REQUESTED** [review]

File: doc/source/publications.rst (hunk 6 above)
Comment (mariaboerner1987) [reply doc/source/publications.rst:13]: Do you have a special paper in your mind? I could only find papers citing Flower together with TFF and PySyft. 
The FjORD is missing but will be added now.

Comment (danieljanes) [reply doc/source/publications.rst:13]: Let's add the FjORD one for now

**Comment by tanertopal** [comment]
@mariaboerner1987 should we also mention: https://www.inovex.de/de/blog/federated-learning-frameworks-part-2/

**Comment by danieljanes** [comment]
@mariaboerner1987 please add the following posts as well:
- https://towardsdatascience.com/scaling-flower-with-multiprocessing-a0bc7b7aace0
- https://towardsdatascience.com/differentially-private-federated-learning-with-flower-and-opacus-e14fb0d2d229

**Comment by mariaboerner1987** [comment]
@tanertopal @danieljanes 
We decided to close this PR and create a separate repository.

**Pull request closed without merging**